constraints/src/main/java/gov/nasa/jpl/aerie/constraints/json/ConstraintParsers.java

```diff
@@ -186,9 +186,10 @@ static JsonParser<Rate> rateF(final JsonParser<Expression<LinearProfile>> linear
 
   static final JsonParser<Duration> durationP =
       longP
-          . map(
-              microseconds -> Duration.of(microseconds, Duration.MICROSECONDS),
-              duration -> duration.in(Duration.MICROSECONDS));
+          .map(
```

Comment:
I think this change ended up being spurious - looks like in the trip through ISO8601 and back we lost the `S`s at the end of `MICROSECOND`.

Reply:
I put the ~s's~ back
I put the ~~esses~~
the ~Ses~

I undid the change